### DISCUSSION TITLE
Unable to save server filters settings

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#13874] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

server filters get reset

### Reproduction steps

1. adjust your server filters
2. restart the game
3. server filters become default

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I created [a ticket about this in the issue tracker](https://github.com/FakeFishGames/Barotrauma/issues/13878), we'll look into it!

**Comment by Rokvach** [comment]
The issue has been fixed and tested to be working, closing discussion.

**Reply by A2ur3** [reply]
it seems the issue is not fully resolved. I still have difficulty saving server tag filter
